### DISCUSSION TITLE
Feedback on chatVariableResolver proposed API

### DISCUSSION THREAD

**Discussion by marechaux** [microsoft/vscode-discussions#1080] [Extension Development QnA]
Hello,

I've been evaluating the `chatVariableResolver` proposed API and appreciate it. 

However, I believe it could be further improved by adding the ability to use a [Quick Pick](https://code.visualstudio.com/api/ux-guidelines/quick-picks) interface for variable selection, similar to the functionality seen with built-in resolvers like `#file`.

**Comment by isidorn** [comment]
Thanks for the feedback, @burkeholland felt the same way

fyi @roblourens

**Comment by abrilurena** [comment]
Hi, @marechaux! thank you so much for sharing your feedback. We truly appreciate it. If there's anything else you would like to share with us or if you need any help - please feel free to contact us ✨ 💯

**Comment by abrilurena** [comment]
Hi, @marechaux! it's been a while since last time, so i'm cosing this issue for now. Please feel free to re-open if you need more help ✨
